src/metabase/api/collection.clj

```diff
@@ -151,10 +151,9 @@
     (t2/hydrate collections :can_write :is_personal :can_delete)
     ;; remove the :metabase.models.collection.root/is-root? tag since FE doesn't need it
     ;; and for personal collections we translate the name to user's locale
-    (for [collection collections]
-      (-> collection
-          (dissoc ::collection.root/is-root?)
-          collection/personal-collection-with-ui-details))))
+    (collection/personal-collections-with-ui-details  (for [collection collections]
+                                                        (-> collection
```

Comment:
now that there's just one following form it feels a little silly to have the `->`, I'd just do `(dissoc collection ::collection.root/is-root?)`